packages/contracts-bedrock/contracts/L2/L2CrossDomainMessenger.sol

```diff
@@ -17,12 +17,12 @@ import { L2ToL1MessagePasser } from "./L2ToL1MessagePasser.sol";
  */
 contract L2CrossDomainMessenger is CrossDomainMessenger, Semver {
     /**
-     * @custom:semver 1.1.0
+     * @custom:semver 1.2.0
      *
      * @param _l1CrossDomainMessenger Address of the L1CrossDomainMessenger contract.
      */
     constructor(address _l1CrossDomainMessenger)
-        Semver(1, 1, 0)
```

Comment:
I just realized that we forgot to bump semver on #5444. Do we need to bump this up twice to `1.3.0` to account for that? 

@tynes I think you have the clearest ideas about that.